opentracing-jaxrs2-itest/apache-cxf/src/test/java/io/opentracing/contrib/jaxrs2/itest/cxf/ApacheCXFParentSpanResolutionTest.java

```diff
@@ -5,6 +5,11 @@
 
 public class ApacheCXFParentSpanResolutionTest extends AbstractParentSpanResolutionTest {
 
+  @Override
+  protected boolean shouldUseParentSpan() {
+    return true;
+  }
```

Comment:
Each of the existing IT classes is basically duplicated for the additional, with the `shouldUseParentSpan()` method controlling whether or not the new opt-in parameter is set, and what the assertion looks for.